src/AlgebraicGeometry/Surfaces/K3Auto.jl

```diff
@@ -2040,3 +2044,66 @@ function has_zero_entropy(S::ZZLat; rank_unimod=26)
 
   return maximum(push!([sign(i) for i in d],-1)), data, K3Autgrp, chambers, rational_curves
 end
+
+  
+function _get_pos_entropy_element(gensAutL)
+  G = [first(gensAutL)^0]
+  f = first(gensAutL)^0
+  n = nrows(f)
+  n<= 4 || error("not implemented")
+  while true
+    Gnew = []
```

Comment:
```suggestion
    Gnew = typeof(f)[]
```
for type stability... resp. if I understand the code right...
```suggestion
    Gnew = ZZMatrix[]
```